dtls_debug.h

```diff
@@ -128,7 +129,7 @@ void dtls_dsrv_log_addr(log_t level, const char *name, const session_t *addr);
 #define dtls_notice(...) LOG_INF(__VA_ARGS__)
 #define dtls_info(...) LOG_INF(__VA_ARGS__)
 #define dtls_debug(...) LOG_DBG(__VA_ARGS__)
-#define dtls_debug_hexdump(name, buf, length) { LOG_DBG("%s (%zu bytes):", name, length); LOG_HEXDUMP_DBG(buf, length, name); }
+#define dtls_debug_hexdump(name, buf, length) { LOG_DBG("-= %s (%zu bytes) =-\n", name, length); LOG_HEXDUMP_DBG(buf, length, name); }
```

Comment:
I do not understand what this change is for and why it is in a RIOT-related PR?

Reply:
@boaks As the one managing and using the Zephyr support, please decide on this one.

LGTM.

Reply:
I guess, that is accidentally change here. According the comment below, it should be in the RIOT part.

@benpicco 
Could you please move it to the RIOT lines below?

Reply:
Oops, good catch - I accidentally added the newline to the wrong line